frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.tsx

```diff
@@ -63,7 +63,7 @@ export const DataPermissionsHelp = () => {
                 icon="permissions_limited"
                 iconColor="warning"
                 name={t`Granular`}
-                description={t`The group can view select schemas and tables. Can be combined with SSO user attributes to create Sandboxes that define what data each person can view.`}
+                description={t`The group can view select schemas and tables. Can be combined with SSO user attributes to create row and column security that define what data each person can view.`}
```

Comment:
"row and column security that defines..."

But maybe the whole sentence needs rethinking?

cc @jeff-bruemmer